websites/K/KEXP/presence.ts

```diff
@@ -0,0 +1,67 @@
+const presence = new Presence({ clientId: "1138970637440917504" }),
+	supportedPaths = [
+		"/archive/",
+		"/podcasts/",
+		"/shows/",
+		"/djs/",
+		"/schedule/",
+	],
+	timeElapsed = Math.floor(Date.now() / 1000);
+async function updatePresence() {
+	const strings = await presence.getStrings({
+			pause: "general.paused",
+			play: "general.playing",
+			profile: "general.readingAbout",
+			reading: "general.readingAnArticle",
+			watching: "general.watching",
+		}),
+		currentPagePath = window.location.pathname,
+		presenceData: PresenceData = {
+			startTimestamp: timeElapsed,
+			largeImageKey: "https://i.imgur.com/uzXz274.png",
+			buttons: [{ label: "Check KEXP", url: "https://kexp.org" }],
```

Comment:
Buttons that direct to main pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).